code/edge/src/androidTest/java/com/adobe/marketing/mobile/EdgeFunctionalTests.java

```diff
@@ -1447,12 +1475,15 @@ public void testSendEvent_fatalError() throws InterruptedException {
 		assertNetworkRequestCount();
 		assertExpectedEvents(false);
 
+		List<Event> requestEvents = getDispatchedEventsWith(EventType.EDGE, EventSource.REQUEST_CONTENT);
+		assertEquals(1, requestEvents.size());
+
 		List<Event> resultEvents = getDispatchedEventsWith(EventType.EDGE, EventSource.ERROR_RESPONSE_CONTENT);
 		assertEquals(1, resultEvents.size());
 
 		Map<String, String> eventData = FunctionalTestUtils.flattenMap(resultEvents.get(0).getEventData());
 
-		assertEquals(10, eventData.size());
+		assertEquals(11, eventData.size());
```

Comment:
processResponseOnError attached the requestEventId even before, is this change for the requestId? I did not find it in the asserts below

Reply:
Sounds good, let's assert for requestId too if we are not doing already.